sark/code/function.py

```diff
@@ -181,6 +181,14 @@ def crefs_to(self):
         return idautils.CodeRefsTo(self.startEA, 1)
 
     @property
+    def calls(self):
+        """
+        Calls to other functions.
+        Returns: Xref objects
+        """
+        return ifilter(lambda x: x.type.is_call, self.xrefs_from)
```

Comment:
It is a generator-expression so it creates a generator, not a list.

**Docs:**
- https://www.python.org/dev/peps/pep-0289/
- https://docs.python.org/2/reference/expressions.html#generator-expressions